src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -111,6 +117,17 @@ private static String getLocalAppDataLocation() {
     return possiblePaths;
   }
 
+  @VisibleForTesting
+  static String unquote(String path) {
+    if (path.startsWith("\"")) {
```

Comment:
Isn't it safer to combine the two conditions?

`if (path.startsWith("\"") && path.endsWith("\""))`